src/dfm/common/data_cleaning/dolma_taggers/language_scandi.py

```diff
@@ -0,0 +1,202 @@
+"""
+
+Filters.
+
+"""
+from collections.abc import Iterable
+
+import pycld2 as cld2
+import regex
+from anyascii import anyascii
+from dolma.core.data_types import DocResult, Document, Span, TextSlice
+from dolma.core.ft_tagger import BaseFastTextTagger, Prediction
+from dolma.core.registry import TaggerRegistry
+from dolma.core.taggers import BaseTagger
+from dolma.core.utils import split_paragraphs
+
+LANGS = {
+    "ENGLISH": "en",
+    "DANISH": "da",
+    "SWEDISH": "sv",
+    "NORWEGIAN": "no",
+    "ICELANDIC": "is",
+    "FAROESE": "fo",  # Note that FAROESE is not supported by cld2
+}
+
+
+@TaggerRegistry.add("cld2_scandi_doc")
+class Cld2LanguageFilterScandi(BaseTagger):
```

Comment:
A small docstring to each of these would be great

Reply:
I have added docstrings to each class in a new commit.